frontend/src/APIClients/GenreAPIClient.ts

```diff
@@ -0,0 +1,25 @@
+import { Genre, Option } from "../types/BookTypes";
+import { getBearerToken } from "../utils/AuthUtils";
+import baseAPIClient from "./BaseAPIClient";
+
+const getGenreOptions = async (): Promise<Option[]> => {
+  try {
+    const { data } = await baseAPIClient.get("/genres", {
+      headers: { Authorization: getBearerToken() },
+    });
+
+    const options: Option[] = [];
+    data.forEach((genre: Genre) => {
+      options.push({
+        value: genre.name,
+        label: genre.name,
+      });
+    });
+
+    return options;
+  } catch (error: unknown) {
+    return error as Option[];
```

Comment:
Instead of catching the error, could you throw it instead? (ie remove the try catch). This will suppress the error and we won't know if something goes wrong

Reply:
On error, maybe display an error toast and set to an empty array (wherever this is called)